tests/core/full_node/test_mempool.py

```diff
@@ -772,15 +758,18 @@ async def test_correct_my_id(self, bt, two_nodes_mempool, wallet_a):
         assert status == MempoolInclusionStatus.SUCCESS
 
     @pytest.mark.asyncio
-    async def test_my_id_garbage(self, bt, two_nodes_mempool, wallet_a):
+    async def test_my_id_garbage(self, bt, one_node_one_block, wallet_a):
 
-        full_node_1, full_node_2, server_1, server_2 = two_nodes_mempool
+        full_node_1, server_1 = one_node_one_block
+
+        _ = await next_block(full_node_1, wallet_a, bt)
+        _ = await next_block(full_node_1, wallet_a, bt)
```

Comment:
you add these extra two blocks in a bunch of places. I *think* we could just change what the test expects instead, but I might be forgetting something

Reply:
The problem is that there are many many tests, so the ones that require the 3 blocks are probably less than 15% here.